app/services/clients/ledger_client.js

```diff
@@ -8,6 +8,44 @@ const defaultOptions = {
   service: 'ledger'
 }
 
+const legacyConnectorTransactionParity = (transaction) => {
+  if (transaction.refund_summary && transaction.refund_summary.amount_refunded) {
+    transaction.refund_summary.amount_submitted = transaction.refund_summary.amount_refunded
+  }
+  transaction.charge_id = transaction.transaction_id
+  return transaction
+}
+
+const transaction = function transaction (id, gatewayAccountId, options = {}) {
+  const configuration = Object.assign({
+    url: `/v1/transaction/${id}`,
+    qs: { account_id: gatewayAccountId },
```

Comment:
it will be changed to `gateway_account_id` so it might be worth adding this one as well as query parameter (later - after the actual changes the `account_id` will be removed)

Reply:
I would rather be explicit? It seems strange to commit multiple possible options to master, this will have to be changed when `account_id` is removed, I'll add a reminder to the ticket

Reply:
The order will need to be anyway: add a new one and then remove the old, otherwise it would break the functionality.